test-extra/Makefile

```diff
@@ -22,6 +22,10 @@ FIND_ARGS= \
 	-path code/ocamlformat/test -not -prune -or \
 	-path code/ocaml/experimental -not -prune -or \
 	-path code/ocaml/testsuite/tests/parse-errors -not -prune -or \
+	-path code/dune/test -not -prune -or \
+	-path code/dune/vendor -not -prune -or \
+	-path code/dune/otherlibs -not -prune -or \
+	-path code/dune/example -not -prune -or \
```

Comment:
Exclude some directory because they contain project-like directory that do not have a `.ocamlformat` file. Or because of parsing errors (`vendor/re` contains unattached doc comments).